src/preferences/dialog/dlgpreflibrarydlg.ui

```diff
@@ -235,24 +242,34 @@
         </property>
        </widget>
       </item>
-      <item row="3" column="1">
+      <item row="4" column="1">
        <widget class="QLineEdit" name="libraryFont">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
-      <item row="3" column="2">
+      <item row="4" column="2">
        <widget class="QToolButton" name="libraryFontButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
-      <item row="4" column="0" colspan="2">
-       <widget class="QCheckBox" name="checkBoxEditMetadataSelectedClicked">
+      <item row="5" column="0">
+       <widget class="QLabel" name="incrementalSearchDelayLabel">
         <property name="text">
-         <string>Edit metadata after clicking selected track</string>
+         <string>Incremental Search Delay:</string>
```

Comment:
This might be misleading for me since "Incremental Search" for me is more an algorithm that works all the time in background guessing the final user result. Google has it in the search bar if the network bandwidth is good enough. 
That would be great for Mixxx, but we are not there.   

"Search after edit" 
"Search As You Type Delay"
Or inverse 
"Search inhibit time"

"Zero disables the feature" ??

Reply:
Ok, I will revert back to the _debouncing_ terminology also for the configuration setting for consistency.

For the text on the label we can use one of the proposed variants. Feedback by native speakers and especially non-developers welcome ;)

Reply:
I don't want to allow a configurable timeout of 0 to disable this feature entirely. A maximum delay of almost 10 sec should be ok. But I've fixed the code to handle empty or invalid timeouts by not starting the timer.